addon/components/frost-toggle.js

```diff
@@ -0,0 +1,113 @@
+import Ember from 'ember'
+const {Component, ViewUtils} = Ember
+import computed from 'ember-computed-decorators'
+import {PropTypes} from 'ember-prop-types'
+import layout from '../templates/components/frost-toggle'
+import FrostEventsProxy from '../mixins/frost-events-proxy'
+
+export default Component.extend(FrostEventsProxy, {
```

Comment:
Can you also add `PropTypeMixin` to this as the default props won't be applied in integration tests of a consumer without it.